ginko/src/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <h1 className="App-title">Ginko Family Tree</h1>
+        </header>
+        <p className="App-intro">
+          This is our app!
+        </p>
+      </div>
+    );
+  }
+}
+
+export default App;
```

Comment:
Is there a way that we can bundle all of the `html` `css` and `js` for each component into a single file?

Also, it would be cool to name each component with the convention of `*.component.js`or something along those lines